main.js

```diff
@@ -0,0 +1,18 @@
+var url = "http://api.icndb.com/jokes/random";
+getJoke()
+var button = document.getElementById("get-joke");
+button.addEventListener("click", function(){
+    getJoke();
+});
+
+var paragraph = document.getElementById("joke");
+
+function getJoke(){
+    var xhr = new XMLHttpRequest();
+    xhr.open("GET", url);
+    xhr.addEventListener("load", function(){
+        var response = JSON.parse(xhr.response);
```

Comment:
at first always check your response and objects inside, you want `value` inside `response` and `joke` inside `value` - check thats keys exist in response ;)

Reply:
I hope you meant including kind of "try-catch" clause :)